schemas/blocks/grand_calculated_summary.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$id": "/blocks/grand_calculated_summary.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/identifier"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["GrandCalculatedSummary"]
```

Comment:
I've done the same thing in my validator changes - using a single value array for enum - just wondering is this the done thing, should we be putting only a single value, would that work?

Reply:
Yeah its been done like this throughout and looks like its the standard for ajv https://ajv.js.org/json-type-definition.html#enum-form so I think thats what it is expecting even if usually just the one type